src/sdk.ts

```diff
@@ -214,6 +214,11 @@ export default class CegaEvmSDK {
     return oracle.description();
   }
 
+  async getOracleDecimals(oracleName: string): Promise<string> {
+    const oracle = await this.loadOracleContract(oracleName);
+    return oracle.decimals();
```

Comment:
seems like `decimals` is a `uint8`. Does that return a `string` or a `BigNumber`?

<img width="508" alt="Screen Shot 2023-07-13 at 10 28 08 AM" src="https://github.com/cega-fi/cega-sdk-evm/assets/1671716/0079f115-30ca-419f-91f8-9da41e43672e">

Reply:
ah it should be a number